gym/vector/async_vector_env.py

```diff
@@ -250,15 +218,13 @@ def reset_wait(
         self,
         timeout: Optional[Union[int, float]] = None,
         seed: Optional[int] = None,
-        return_info: bool = False,
         options: Optional[dict] = None,
     ) -> Union[ObsType, Tuple[ObsType, List[dict]]]:
         """Waits for the calls triggered by :meth:`reset_async` to finish and returns the results.
 
         Args:
             timeout: Number of seconds before the call to `reset_wait` times out. If `None`, the call to `reset_wait` never times out.
             seed: ignored
```

Comment:
The seed is ignored, how to vector env seed their environments without `seed`

Reply:
So I looked into this, basically reset_wait doesn't use its arguments, since seeding is performed in reset_async, so I think we can safely remove the arguments in reset_wait for vector envs. I think this should be fixed in a separate PR though.